.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 bin/
 build/
+*nabla
```

Comment:
Is this for just `.nabla` files? Should this be `*.nabla` if that's the case?

Reply:
Ya, that's right